doc/api_assets/style.css

```diff
@@ -79,6 +79,15 @@ code a:hover {
   color: white !important;
 }
 
+pre.api_stability_0 a,
+pre.api_stability_1 a,
+pre.api_stability_2 a,
+pre.api_stability_3 a,
+pre.api_stability_4 a,
+pre.api_stability_5 a {
```

Comment:
Wait, I think it could cause side effects, so your approach might be better overall. Let's just remove the overqualified selectors, so it's just `.api_stability_0 a` etc.